ow_regolith/include/RegolithSpawner.h

```diff
@@ -74,7 +74,7 @@ class RegolithSpawner
   // regolith will spawn once this amount of volume is displaced
   double m_spawn_threshold;
   // Gazebo link name of the scoop particles will spawn in
-  std::string m_scoop_link_name;
+  std::string m_scoop_linkname;
```

Comment:
I think **m_scoop_link_name** is the conformant style, but anyway
```suggestion
  std::string m_scoop_link_name;
```